lang/julia/Portfile

```diff
@@ -22,124 +22,82 @@ long_description    Julia is a high-level, high-performance dynamic programming
 
 github.tarball_from releases
 
-distfiles           ${name}-${version}-full${extract.suffix}
+distfiles           ${name}-${version}-full${extract.suffix} \
+                    ${name}-${version}-full${extract.suffix}.asc
```

Comment:
What is the `.asc` file for? Is it just a signature? Is it used for anything?

Reply:
Yes, the `.asc` is a gpg signature for the tarball.

Reply:
It's used to verify the tarball. More on this below.